_data/form.yml

```diff
@@ -0,0 +1,831 @@
+      type: number
+
+  - group: proposed-number-buildings
+    fields:
+    - label: Proposed number of buildings
+      type: number
+
+  - group: proposed-number-stories
+    fields:
+    - label: Proposed number of stories
+      type: number
+
+  - group: proposed-number-basements
+    fields:
+    - label: Proposed number of basements or cellars
+      type: number
+
+  - group: proposed-parking-spaces
+    fields:
+    - label: Proposed parking spaces
+      type: number
+
+  - group: proposed-bicycle-spaces
+    fields:
+    - label: Proposed bicycle spaces
+      type: number
+
+- title: Fire protection
+  fields:
+  - label: Does the property have fire sprinklers?
+    type: radio
+    options:
+    - label: "Yes"
+      shows: sprinklers
+
+    - label: "No"
+      hides: sprinklers
+
+  - group: sprinklers
+    fields:
+    - label: Which areas of the property do the sprinklers cover?
+
+  - label: Does the property have fire alarms?
+    type: radio
+    options:
+    - label: "Yes"
+      shows: alarms
+
+    - label: "No"
+      hides: alarms
+
+  - group: alarms
+    fields:
+    - label: Which areas of the property do the alarms cover?
+
+- title: Worker's compensation insurance
+  fields:
+  - label: What best describes your worker's compensation insurance coverage for this project?
+    type: radio
+    options:
+    - label: "I am self-insured and have a certificate of consent."
+      hides: comp-II
+      shows: comp-I
+
+    - label: "I have third-party insurance."
+      shows: comp-II
+
+    - label: "I plan to hire a contractor who will follow worker's compensation laws."
+      hides: comp-II
+      shows: comp-V
+
+    - label: "I don't need insurance because I'm not hiring anyone."
+      hides: comp-II
+      shows: comp-IV
+
+    - label: "I don't need insurance because the work will cost under $100."
+      hides: comp-II
+      shows: comp-III
+
+  - group: comp-II
+    fields:
+    - label: Name of insurance carrier
+
+    - label: Policy number
+      type: number
+
+- title: Seismic work
+  fields:
+  - label: >
+      Have you filed for mandatory seismic retrofitting under [Chapter 4D of the Existing Building Code](http://library.amlegal.com/nxt/gateway.dll/California/sfbuilding/existingbuildingcode2016edition/chapter4dmandatoryearthquakeretrofitofwo?f=templates$fn=default.htm$3.0)?
+    type: radio
+    options:
+    - label: "Yes"
+      shows: seismic-retro-mandatory
+      hides: seismic-retro-number
+
+    - label: "No"
+      shows: seismic-retro-mandatory
+      hides: seismic-retro-number
+
+  - group: seismic-retro-number
+    fields:
+    - label: Enter your seismic retrofitting permit application number.
+      type: number
+
+  - label: >
+      Have you filed for voluntary seismic strengthening under [Administrative Bulletin AB-094](https://sfdbi.org/sites/default/files/migrated/FileCenter/Documents/Administrative_Bulletins/2013_AB/AB_094_updated_010114.pdf)?
```

Comment:
Persis: Try linking out to the ordinances in the help text, not the label.

Reply:
If the applicant has done these things, they likely know the answer already. So you don't necessarily need specialist language (like citing the ordinance.)

We should have a standard design pattern for this.